web/ts/watch.ts

```diff
@@ -18,6 +18,34 @@ let ws: WebSocket;
 let retryInt = 5000; //retry connecting to websocket after this timeout
 const pageloaded = new Date();
 
+function initChatScrollListener() {
+    const chatBox = document.getElementById("chatBox") as HTMLDivElement;
+    if (chatBox === undefined || chatBox === null) {
```

Comment:
I think this could be `if (!chatBox) {...}` for a bit more generic check, but this is a matter of taste